File: practias_logica_programacion/python_practicas/ejercicio7_cantidadImpares.py


def cantidad_impares(num1, num2):

    list_num = []
    for num in range(num1, num2):
        if num%2 != 0:
            list_num.append(num)
    print((f'La cantidad de numeros impares que hay entre {num1} y {num2} es de {len(list_num)}'))
    cantidad_impares(1,10)

File: practias_logica_programacion/javaScript_practicas/ejercicio7_cantidadImpares.js

'use strict'


function cantidad_impares(num1,num2){

    let save_array = []
    for(let i =num1; i<(num2+1); i++){
        if(i%2 != 0){
            save_array.push(i)
        }
    }
    console.log(`la cantidad de impares que hay entre ${num1} y ${num2} es ${save_array.length}`)
}

cantidad_impares(1,10)